do.js

```diff
@@ -0,0 +1,68 @@
+const Account = require('./account/do-account')
+const Droplets = require('./compute/do-droplets')
+const LoadBalancers = require('./networking/do-load-balancers')
+const FloatingIps = require('./networking/do-floating-ips')
+const Volumes = require('./storage/do-volumes')
+
+class DO {
+	/**
+   * Expose DigitalOcean APIs
+   * @constructor
+   */
+  constructor(token,doSDK) {
+		this._doToken = token;
+		this._DO = doSDK;
+
+		return {
+			getSDK: () => this._DO,
+      getToken: () => this._doToken,
+      account: this.Account,
+      droplets: this.Droplets,
+      loadbalancers: this.LoadBalancers,
+      floatingips: this.FloatingIps,
+      volumes: this.Volumes
+		};
+  }
+
+  /**
+   * Account - Account & SSH Wrapper
+   * @Account
+   */
+  Account() {
+    return new Account(this.getSDK(), this.getToken());
+  }
+
+	/**
+   * Compute - Droplets Wrapper
+   * @Droplets
+   */
+  Droplets() {
+    return new Droplets(this.getSDK(), this.getToken());
+  }
+
+  /**
+   * Networking - Load Balancers Wrapper
+   * @LoadBalancers
+   */
+  LoadBalancers() {
+    return new LoadBalancers(this.getSDK(), this.getToken());
+  }
+
+  /**
+   * Networking - Floating IPs Wrapper
+   * @FloatingIps
+   */
+  FloatingIps() {
```

Comment:
Change method names to camelCase

Reply:
I've followed camelCase throughout the codebase with the wrapper classes as an exception. These method names belong to the wrapper classes. Should we necessarily also change them to camelCase?